cray_product_catalog/schema/schema.yaml

```
@@ -147,4 +151,26 @@ definitions:
         description: The type of repository
         type: string
         enum:
-          - group
\ No newline at end of file
+          - group
+  S3Repositories:
+    description: An array of artifact repositories.
+    type: array
+    items:
+      required:
+        - bucket
+        - key
+      description: A component with a bucket and key.
+      type: object
+      properties:
+        name:
+          bucket: string
+          description: The bucket name.
+        version:
+          key: string
+          description: The key.
```

This looks wrong to me. Shouldn't it be?
```suggestion
        bucket:
          type: string
          description: The S3 bucket name.
        key:
          type: string
          description: The key to the artifact in the specified S3 bucket.
```